File: Example/CYPhotoLibrary/UI/Model/CYPhoto.m
//
//  CYPhoto.m
//  CYPhotoKit
//
//  Created by dongzb on 16/3/20.
//  Copyright © 2016年 大兵布莱恩特. All rights reserved.
//

#import "CYPhoto.h"
#import "CYPhotosAsset.h"

@implementation CYPhoto

- (void)setPhotosAsset:(CYPhotosAsset *)photosAsset {
    _photosAsset = photosAsset;
    self.asset   = _photosAsset.asset;
}
@end

File: Example/CYPhotosKit/Model/CYPhoto.swift
//
//  CYPhoto.swift
//  BayeSocial
//
//  Created by 董招兵 on 2017/3/31.
//  Copyright © 2017年 大兵布莱恩特. All rights reserved.
//

import UIKit
import Photos

enum CYPhotoAssetType : Int {
    case add = 1
    case photo
}

class CYPhoto : NSObject {
    var image : UIImage?
    var asset : PHAsset?
    var photosAsset : PHAsset? {
        didSet {
            self.asset = photosAsset
        }
    }
    var type : CYPhotoAssetType = .add
    override init() {
        
    }
}
